Tests/SwiftParserTest/StatementTests.swift

```diff
@@ -265,16 +279,32 @@ final class StatementTests: ParserTestCase {
     assertParse(
       """
       func test1() {
-        1️⃣@s return
+        @s 1️⃣return
       }
       func test2() {
-        2️⃣@unknown return
+        @unknown 2️⃣return
       }
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code '@s' before 'return' statement"),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "unexpected code '@unknown' before 'return' statement"),
-      ]
+        DiagnosticSpec(
+          locationMarker: "1️⃣",
+          message: "expected declaration and ';' after attribute",
+          fixIts: ["insert declaration and ';'"]
```

Comment:
Why do we default to `;` instead of a newline for this case?

Reply:
This is modeled as
```
CodeBlockItem(
  item: .decl(MissingDecl(...)),
  semicolon: .semiColonToken(precence: .missing)
)
```
And `ParseDiagnosticsGenerator` generates single diagnostic for consecutive missing nodes.

"consecutive statements ... insert newline or ';'" is a special diagnostics that is generated when the previous node was not missing.